src/framework/handlers/css.js

```diff
@@ -8,6 +8,14 @@ class CssHandler {
      */
     handlerType = "css";
 
+    /**
+     * TextDecoder for decoding binary data.
+     *
+     * @type {TextDecoder}
+     * @private
+     */
+    decoder = new TextDecoder('utf-8');
```

Comment:
Is there a solution here for older devices without TextDecoder?

Reply:
Which devices would that be in particular?
It has great support (97.8%): https://caniuse.com/textencoder
Only devices with software that has not been updated since 2016/18 potentially will not support it, but that is like 6-8 year outdated.
IE11 is not supported anymore already by the engine.

Reply:
The UMD build is still [targeting some fairly old browsers](https://github.com/playcanvas/engine/blob/53d2ce4824453c0eee3801dd417ba3bceb7791b1/utils/rollup-es5-options.mjs#L22-L24). It's reasonable to require a recent browser to use bundles, but the engine probably shouldn't throw an error if not. Could we lazily initialize so it only happens when a bundle is loaded?

Reply:
Pretty sure you can easily polyfill TextDecoder such as https://stackoverflow.com/questions/40662142/polyfill-for-textdecoder